content/en/docs/tutorials/acme/ingress.md

```diff
@@ -222,30 +221,16 @@ application which makes an excellent back-end for examples.
 The quick-start example uses three manifests for the sample. The first two are a
 sample deployment and an associated service:
 
-TODO (`@joshvanl`): move this link to the new location
-- deployment manifest:
-  [`deployment.yaml`](https://raw.githubusercontent.com/jetstack/cert-manager/release-0.12/docs/tutorials/acme/quick-start/example/deployment.yaml)
-
-TODO (`@joshvanl`): include link contents
-```yaml
-example/deployment.yaml
-```
-
-TODO (`@joshvanl`): move this link to the new location
-- service manifest:
-  [`service.yaml`](https://raw.githubusercontent.com/jetstack/cert-manager/release-0.12/docs/tutorials/acme/quick-start/example/service.yaml)
+{{% include file="/content/en/docs/tutorials/acme/example/deployment.yaml" language="yaml" %}}
 
-TODO (`@joshvanl`): include link contents
-```yaml
-example/service.yaml
-```
+{{% include file="/content/en/docs/tutorials/acme/example/service.yaml" language="yaml" %}}
 
 You can create download and reference these files locally, or you can
 reference them from the GitHub source repository for this documentation.
 To install the example service from the tutorial files straight from GitHub,
 you may use the commands:
 
-```yaml
+```bash
 $ kubectl apply -f https://raw.githubusercontent.com/jetstack/cert-manager/release-0.12/docs/tutorials/acme/quick-start/example/deployment.yaml
```

Comment:
I'm slightly nervous that we now have to maintain and keep in sync both versions of this file - is there anything we can do here to avoid this?

Reply:
Why are we maintaining two versions?